Sources/BitcloutIdentity/Models/EncryptedMessagesThread.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  EncryptedMessages.swift
+//  
+//
+//  Created by Andy Boyd on 29/06/2021.
+//
+
+import Foundation
+
+public struct EncryptedMessagesThread: Equatable {
+    let publicKey: String
+    let encryptedMessages: [String]
```

Comment:
Is the identity library responsible for storing encrypted and/or decrypted messages?

Reply:
No, the app will be responsible for storing them. They'll just be passed in encrypted form to the library, and then they'll be decrypted and sent back to the app. How they're stored is a client implementation detail beyond the scope of the library's responsibilities.